feature-utils/poly-look/src/react-components/filterChips/chip.jsx

```diff
@@ -13,11 +14,18 @@ const Chip = ({ id, translation, handleClick, active }) => {
   return (
     <button
       className={active ? "chip selected" : "chip"}
-      onClick={handleClick ? () => handleClick(id) : () => {}}
+      onClick={() => (handleClick ? handleClick(id) : {})}
```

Comment:
Is this the result of linting? There's not really a big difference, right?

Reply:
It is neither the result of the linting nor does it do anything new. I previously made a change to set handleClick's default value to `() => {}`, but it resulted in an error, so I changed it back.

Even though the new "change" is doing the same, I'll change it back to **exactly** how it was.